mrp_production_request/models/mrp_production_request.py

```diff
@@ -108,6 +113,19 @@ def _compute_manufactured_qty(self):
                    ("cancel", "Cancelled")],
         index=True, track_visibility='onchange',
         required=True, copy=False, default='draft')
+    procurement_group_id = fields.Many2one(
+        string='Procurement Group',
+        comodel_name='procurement.group',
+        copy=False)
+    procurement_ids = fields.One2many(
+        string='Related Procurements',
+        comodel_name='procurement.order',
+        inverse_name='mrp_production_request_id')
+    propagate = fields.Boolean(
```

Comment:
Why do you add these three fields? They aren't used anywhere, are they?

Reply:
Hi, Thanks for your review!
I did this PR to fix warnings in runbot, the absence of these 3 fields generated [this warning](https://travis-ci.org/OCA/manufacture/jobs/402942342#L1340) because the procurement order creates the `mrp_production_request` in [this way](https://github.com/SimoRubi/manufacture/blob/2c743e97f1774a4d994ae71d6fad3f5a4493af65/mrp_production_request/models/procurement.py#L18): among others it gives a value to these 3 of fields that, before this PR, were in `mrp.production` but not in `mrp.production.request`.

Reply:
Ok, I got you. Then, please add those new fields in [`_prepare_manufacturing_order`](https://github.com/SimoRubi/manufacture/blob/2c743e97f1774a4d994ae71d6fad3f5a4493af65/mrp_production_request/wizards/mrp_production_request_create_mo.py#L84) method.

Reply:
Done in https://github.com/OCA/manufacture/pull/288/commits/f1273cba85ae97c0f11e742a964c942e2796a273